{
  "cve": "CVE-2023-6247",
  "mitre": {
    "cpes": [],
    "created": "2024-02-20T18:43:17.890000+00:00",
    "description": "The PKCS#7 parser in OpenVPN 3 Core Library versions through 3.8.3 did not properly validate the parsed data, which would result in the application crashing.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2023/6xxx/CVE-2023-6247.json",
    "references": [
      "https://community.openvpn.net/openvpn/wiki/CVE-2023-6247"
    ],
    "title": null,
    "updated": "2024-08-02T08:21:18.110000+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-476"
    ]
  },
  "nvd": {
    "cpes": [],
    "created": "2024-02-29T01:42:34.380000+00:00",
    "description": "The PKCS#7 parser in OpenVPN 3 Core Library versions through 3.8.3 did not properly validate the parsed data, which would result in the application crashing.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2023/CVE-2023-6247.json",
    "references": [
      "https://community.openvpn.net/openvpn/wiki/CVE-2023-6247"
    ],
    "title": null,
    "updated": "2024-02-29T13:49:47.277000+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-476"
    ]
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [],
      "providers": []
    },
    "created": {
      "data": "2024-02-20T18:43:17.890000+00:00",
      "provider": "mitre"
    },
    "description": {
      "data": "The PKCS#7 parser in OpenVPN 3 Core Library versions through 3.8.3 did not properly validate the parsed data, which would result in the application crashing.",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {},
        "provider": null
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": null,
        "provider": null
      }
    },
    "references": {
      "data": [
        "https://community.openvpn.net/openvpn/wiki/CVE-2023-6247"
      ],
      "providers": [
        "mitre",
        "nvd",
        "vulnrichment"
      ]
    },
    "title": {
      "data": null,
      "provider": null
    },
    "updated": {
      "data": "2024-08-02T08:21:18.110000+00:00",
      "provider": "mitre"
    },
    "vendors": {
      "data": [],
      "providers": []
    },
    "weaknesses": {
      "data": [
        "CWE-476"
      ],
      "providers": [
        "mitre",
        "nvd"
      ]
    }
  },
  "vulnrichment": {
    "cpes": [],
    "created": "2024-02-20T18:43:17.890000+00:00",
    "description": "The PKCS#7 parser in OpenVPN 3 Core Library versions through 3.8.3 did not properly validate the parsed data, which would result in the application crashing.",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {},
      "kev": {},
      "ssvc": {}
    },
    "references": [
      "https://community.openvpn.net/openvpn/wiki/CVE-2023-6247"
    ],
    "title": null,
    "updated": "2024-08-02T08:21:18.110000+00:00",
    "vendors": [],
    "vulnrichment_repo_path": "2023/6xxx/CVE-2023-6247.json",
    "weaknesses": []
  }
}